todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,23 @@
+from todo_app.data.view_model import ViewModel
+from todo_app.data.trello_items import Item
+import pytest
+
+test_cards = [
+    {'id': 1, 'status': 'Done', 'name': 'Test1', 'card_id': 'test_card_id_1', 'list_id': 'done_list_id'},
+    {'id': 2, 'status': 'To Do', 'name': 'Test2', 'card_id': 'test_card_id_2', 'list_id': 'todo_list_id'}
+]
+
+@pytest.fixture
+def view_model() -> ViewModel:
+    test_items = [Item.from_trello_card(card) for card in test_cards]
+    return ViewModel(test_items)
+
+def test_view_model_done_property(view_model: ViewModel):
+    done_items = view_model.done_items
+    for item in done_items:
+        assert item.status == 'Done'
```

Comment:
I think I'd like to see one additional assertion here - this could trivially pass if the `done_items` array just returned nothing, or it could convert all items to done or something weird like that. Much nicer to also assert that the length is what you'd expect:
```suggestion
    for item in done_items:
        assert item.status == 'Done'
    assert len(done_items) == 1
```